docs/docs/the_aztec_network/guides/running_delegated_stake.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 5
+title: Running Delegated Stake
+description: Learn how to run a sequencer with delegated stake on the Aztec network.
+---
+
+## Overview
+
+This guide covers the additional steps required to run a sequencer with delegated stake on the Aztec network. In conventional setups, you must have the required stake to join the sequencer set. In a delegated setup, you (the "provider") use someone else's tokens as stake to back your attestations and proposals rather than your own.
```

Comment:
Can we reword some of this to make it super clear that this a non-custodial staking system?